### PR TITLE
Add mute property into navigation branch

### PR DESCRIPTION
**Pull request by wonsuk73**
It is required to add date property for navigation mute.

### CODE REVIEW

File: spec/Cabin/Infotainment.vspec [hunk 1]
```diff
@@ -89,6 +89,12 @@ Navigation.DestinationSet.Longitude:
   max: 180
   unit: degrees
   description: Longitude of destination in WGS 84 geodetic coordinates.
+  
+Navigation.Mute:
+  datatype: string
+  type: actuator
+  allowed: ['Muted', 'Alerts only', 'Unmuted']
```
Comment (erikbosch) [inline spec/Cabin/Infotainment.vspec:96]: I would prefer if the [style guide](https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/allowed/#recommendation-on-string-values) is followed, i.e. use upper case and underscore. (Alternatively we can discuss the style guide at next meeting to see if we still consider the style guide as relevant)

i.e. something like:

`allowed: ['MUTED', 'ALERT_ONLY', 'UNMUTED']`

**Comment by erikbosch** [comment]
A minor comment - according to https://www.covesa.global/contribute there should be a sign-off on each commit (e.g. `git commit -s`). We have not benn following that rule strictly in the past, but for new commits we shall try to follow it.

File: spec/Cabin/Infotainment.vspec (hunk 1 above)
Comment (wonsuk73) [reply spec/Cabin/Infotainment.vspec]: @erikbosch Thanks for quick comment! I am sorry that It is the same comment with previous my PR :( I have applied your comment.

**Comment by wonsuk73** [comment]
@erikbosch Thanks for this comment. I will follow up the guideline you mentioned.

**Review by erikbosch — APPROVED** [review]
Looks good to me. I will be on vacation next week, I suggest that @adobekan or @SebastianSchildt  initiates the discussion on the meeting next week if this PR can be merged

**Comment by adobekan** [comment]
looks good

**Pull request merged**